test/core/transport/binder/end2end/fuzzers/fuzzer_utils.cc

```diff
@@ -0,0 +1,139 @@
+// Copyright 2021 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test/core/transport/binder/end2end/fuzzers/fuzzer_utils.h"
+
+namespace grpc_binder {
+namespace fuzzing {
+
+std::thread* g_fuzzing_thread = nullptr;
```

Comment:
Can you put this variable in anonymous namespace and provide `CreateFuzzingThread()` `GetFuzzingThread()` `DeleteFuzzingThread()` helper functions to access it?

(It is a bit messy to access global variable defined in other files directly)

Reply:
Or, alternatively, define `g_fuzzing_thread` in client fuzzer / server fuzzer `.cc` file and let caller pass it into `TranasctionReceiverForFuzzing::TranasctionReceiverForFuzzing()`

Reply:
Done.